getting-started/aws/sample-helm/values-velocity.yaml

```diff
@@ -1,13 +1,13 @@
 # {velocity.v1.envName} will be translated to unique ID per environment
 
 # bucket_name is prefixed with a random string and the environment name. This is because buckets are globally unique, so this allows us to have a bucket per environment without collisions.
-bucket_name: '{velocity.v1.generate:random([a-z]{1,64})}-{velocity.v1.envName}-my-bucket'
+bucket_name: 'velocity-{velocity.v1.generate:random([a-z]{1,64})}-{velocity.v1.envName}-my-bucket'
```

Comment:
I think we should include the full template solution when ready instead of this, WDYT?